api/database/factories/ApplicantFilterFactory.php

```diff
@@ -95,20 +95,18 @@ public function withRelationships(bool $sparse = false)
             )->get();
             $filter->pools()->saveMany($pools);
             $filter->qualifiedClassifications()->saveMany($pools->flatMap(fn ($pool) => $pool->classification));
-            $stream = (empty($pools) || count($pools) === 0) ? $this->faker->randomElements(
-                array_column(PoolStream::cases(), 'name'),
-            ) : [$pools[0]->stream];
+            $streams = $pools->map(fn ($pool) => $pool->workStream);
+            $filter->workStreams()->saveMany($streams);
 
             $ATIP = Community::where('key', 'atip')->first();
             $digital = Community::where('key', 'digital')->first();
 
-            if (in_array(PoolStream::ACCESS_INFORMATION_PRIVACY->name, $stream) && $ATIP?->id) {
+            if ($streams->first()?->key === PoolStream::ACCESS_INFORMATION_PRIVACY->name && $ATIP?->id) {
```

Comment:
Shouldn't we be getting rid of PoolStream uses?  Also, why use first() in this collection?

Reply:
Oh, good catch on the first. I have updated it to use contains to get closer to the original condition.

Also, all the references to `PoolStream` should also be fixed in [remove new references to PoolStream](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12222/commits/35e4f95d6e3ba0d1cce7a1eec2825bfd24d89e0b)